### ISSUE TITLE
Choice Active day(s)

### ISSUE THREAD

**Issue by realmahd1** [Kiarash-Z/react-modern-calendar-datepicker#316]
Hi
I like to suggest adding Actived Day(s), just like the functionality of Disabled Day(s), but the user can only pick Actived Day(s).

Thanks!

**Comment by rnwonder** [comment]
How do you mean pick only active days

**Comment by realmahd1** [comment]
Sorry for answering late.

For example, I want the 19th of July and the 22nd of July to be active, and the rest of the days to be disabled.

**Comment by realmahd1** [comment]
Also, the maintainer is not active , I think he won't maintain the package anymore.

**Comment by realmahd1** [comment]
> How do you mean pick only active days

Sorry for answering late.

For example, I want the 19th of July and the 22nd of July to be active, and the rest of the days to be disabled.
Also, the maintainer is not active , I think he won't maintain the package anymore.

**Comment by rnwonder** [comment]
> > How do you mean pick only active days
> 
> Sorry for answering late.
> 
> For example, I want the 19th of July and the 22nd of July to be active, and the rest of the days to be disabled. Also, the maintainer is not active , I think he won't maintain the package anymore.

Yeah it isn't managed anymore, but I built a similar date picker library but its in solidjs not react though. I added this functionality a while ago you can check it out if you ever try using solidjs https://github.com/rnwonder/solid-date-picker
